llms/mlx_lm/utils.py

```diff
@@ -10,16 +10,21 @@
 from transformers import AutoTokenizer, PreTrainedTokenizer
 
 # Local imports
-from .models import llama, phi2
+from .models import llama, mixtral, phi2
 from .models.base import BaseModelArgs
 
 # Constants
 MODEL_MAPPING = {
     "llama": llama,
     "mistral": llama,  # mistral is compatible with llama
+    "mixtral": mixtral,
     "phi": phi2,
 }
 
+linear_class_predicate = (
+    lambda m: isinstance(m, nn.Linear) and m.weight.shape[0] % 32 == 0
```

Comment:
Using the same assertion (0th dimension) in mlx quantize, would go with `all (dim % 32 == 0 for dim in m.weight.shape)`, but it feels a bit overkill.

Reply:
@mzbac just FYI we are adding support for non 32 multiple weights in https://github.com/ml-explore/mlx/pull/458 so we will be able to remove this very soon.

Reply:
> @mzbac just FYI we are adding support for non 32 multiple weights in [ml-explore/mlx#458](https://github.com/ml-explore/mlx/pull/458) so we will be able to remove this very soon.

🚀 🚀 🚀